internal/node/machine_mgr.go

```diff
@@ -134,7 +146,7 @@ func (m *MachineManager) Start() {
 				continue
 			}
 
-			vm, err := createAndStartVM(m.ctx, m.config, m.log)
+			vm, err := createAndStartVM(context.TODO(), m.config, m.log)
```

Comment:
this is a suggestion for the future, not required. I wasn't aware of the todo context's use as a legitimate placeholder.

Reply:
Firecracker was throwing some nasty shutdown errors. It's internal shutdown code seems to be a bit async and since we are shelling out to it I moved us to `TODO` for the time being. I am going to experiment with this some more but this is an improvement in the meantime.